modules/javafx.controls/src/main/java/javafx/scene/control/TextFormatter.java

```diff
@@ -199,7 +199,7 @@ void unbindFromControl() {
     }
 
     void updateValue(String text) {
-        if (!value.isBound()) {
+        if (valueConverter != null &&!value.isBound()) {
```

Comment:
Minor: please add a space between the `&&` and `!` operators.

Reply:
done.